bintree.go

```diff
@@ -15,6 +15,7 @@ const (
 	nodeTypeOr
 	nodeTypeAnd
 	nodeTypeNot
+	nodeTypeNeor
```

Comment:
The normal "or" is already "non-exclusive", in the sense that it's not an exclusive or (xor). What do you think about "non-short-circuiting or", or nssor for short?

Reply:
There must be a term for 'non-short-circuiting', I wonder if we can find it, I agree that the current name sucks though (it also happens to have a bug in it anyways, so cant be merged).

Reply:
`nodeTypeLcor` => `long-circuiting OR` ? :D

Reply:
`nodeTypeUmbrella` ?

Reply:
See #12